frontend/vite.config.mjs

```diff
@@ -17,7 +17,9 @@ const plugins = [
       VuetifyResolver(),
     ],
   }),
-  createSvgPlugin(),
+  svg4VuePlugin({
+      skipSvgo: true
```

Comment:
- What does svgo do? Why do we want to disable it?
- I think the option key should be `svgoConfig`, at least according to the [docs](https://github.com/yisibell/vite-plugin-svg4vue?tab=readme-ov-file#options).